src/templates/partials/caddy.hbs

```diff
@@ -1,43 +1,31 @@
 # {{output.header}}
 # {{{output.link}}}
-{{#unless (includes "old" form.config)}}
+{{#unless (minver "2.0.0" form.serverVersion)}}
+# note that Caddy version 1 reached end-of-life > 4 years ago back in 2020
+{{else}}
```

Comment:
I'd like to do that in a consistent way, as with unsupported tls13 — emit those empty configs similarly. More templates will need that, I'll follow up later.

Reply:
Please add to your wishlist to track there.

Reply:
This actually doesn't stop the output for Caddy 1, only adds a comment and then keep with Caddy 2 config that's not really compatible.

I'll look into all of the above and will co-author the followup PR as some of the changes needed won't be allowed to be submitted as review suggestions et al., so I'll just pick up over there…